src/haz3lweb/view/SchoolMode.re

```diff
@@ -51,6 +55,12 @@ let render_cells = (settings: ModelSettings.t, v: list(vis_marked(Node.t))) => {
   );
 };
 
+/* Takes in the editor information (eds)
```

Comment:
the `view` function doesn't seem to be taking in eds directly, not sure this comment is necessary